buildroot-external/overlay/WebUI/etc/init.d/S50lighttpd

```diff
@@ -82,7 +82,7 @@ reload() {
     echo "include \"/etc/lighttpd/conf.d/proxy_normal.conf\"" >/var/etc/lighttpd_proxy.conf
   fi
 
-  start-stop-daemon -K -s USR1 -q -p ${PIDFILE_ANGEL}
+  start-stop-daemon -K -s HUP -q -p ${PIDFILE_ANGEL}
```

Comment:
I had to revert back to sending `HUP` instead of just `USR1` because when turning on/off the authentication or https redirection lighttpd seem to not have refreshed all things correctly and returned some permission denied or `403 Forbidden` even on port 443. Changing the reload() method to use HUP instead of USR1 immediately solved this issue.